MSAL/src/MSALError_Internal.h

```diff
@@ -79,5 +79,7 @@ extern void MSALFillAndLogError(NSError * __autoreleasing *, id<MSALRequestConte
 
 #define REQUIRED_PARAMETER_ERROR(_PARAMETER, _CTX) MSALFillAndLogError(error, _CTX, MSALErrorInvalidParameter, nil, nil, nil, __FUNCTION__, __LINE__, @#_PARAMETER " is a required parameter and must not be nil or empty.")
 
-// Convenience macro for creating a cache error
-#define MSAL_KEYCHAIN_ERROR_PARAM(_CTX, _OSERROR, _OPERATION) MSAL_OSERROR_PARAM(_CTX, MSALErrorKeychainFailure, _OSERROR, @"Keychain failure (%d) while %@", (int)_OSERROR, _OPERATION)
+// Convenience macro for creating a keychain error
+#define MSAL_KEYCHAIN_ERROR(_CTX, _OSSTATUS) \
```

Comment:
You absolutely need to keep the operation and the coherent keychain logging here so we have some idea what it was doing when it failed. It's also useful to include that string in the user dictionary under the MSALErrorDescriptionKey